README.md

```diff
@@ -1,13 +1,26 @@
 # Qiskit Machine Learning
 
-[![License](https://img.shields.io/github/license/Qiskit/qiskit-machine-learning.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Build Status](https://github.com/qiskit-community/qiskit-machine-learning/actions/workflows/main.yml/badge.svg)](https://github.com/qiskit-community/qiskit-machine-learning/actions?query=workflow%3A"Machine%20Learning%20Unit%20Tests"+branch%3Amain+event%3Apush)[![](https://img.shields.io/github/release/Qiskit/qiskit-machine-learning.svg?style=popout-square)](https://github.com/qiskit-community/qiskit-machine-learning/releases)[![](https://img.shields.io/pypi/dm/qiskit-machine-learning.svg?style=popout-square)](https://pypi.org/project/qiskit-machine-learning/)[![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit-machine-learning/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit-machine-learning?branch=main)<!--- long-description-skip-end -->
-
-Qiskit Machine Learning introduces fundamental computational building blocks - such as Quantum Kernels
-and Quantum Neural Networks - used in different applications, including classification and regression.
+[![License](https://img.shields.io/github/license/qiskit-community/qiskit-machine-learning.svg?)](https://opensource.org/licenses/Apache-2.0) <!--- long-description-skip-begin -->
+[![Current Release](https://img.shields.io/github/release/qiskit-community/qiskit-machine-learning.svg?logo=Qiskit)](https://github.com/qiskit-community/qiskit-machine-learning/releases)
+[![Extended Support Release](https://img.shields.io/github/v/release/qiskit-community/qiskit-machine-learning?sort=semver&filter=0.*&logo=Qiskit&label=extended%20support)](https://github.com/qiskit-community/qiskit-machine-learning/releases?q=tag%3A0)
```

Comment:
My take would be to leave out an extended support shield until such time as it is meaningful. The normal procedure here, so far is that the present (minor at this time) release is supported, by bug fixes, any prior release the support is discontinued. E.g so at at point if 0.8.0 is released then 0.7.x support would cease. When things get to 1.0.0 like Qiskit, that has a bunch of breaking changes so to give users time to migrate they are continuing to (i..e extending) support of 0.46 .

Reply:
Thanks for pointing this out - now this reflects the discussion from the Security policy.